server-phoenix/lib/helios/application.ex

```diff
@@ -7,6 +7,11 @@ defmodule Helios.Application do
 
   @impl true
   def start(_type, _args) do
+    unless Mix.env == :prod do
+      Dotenv.load
```

Comment:
Does this find the `.env.local` file in the parent directory?

Reply:
I'm getting this locally:
```
iex(1)> Dotenv.get("WEATHER_API_KEY")
nil
```

Reply:
That library might only work for .env files, not .env.local